src/main/webapp/app/course/dashboards/instructor-course-dashboard/instructor-course-dashboard.component.ts

```diff
@@ -71,8 +72,8 @@ export class InstructorCourseDashboardComponent implements OnInit {
             tap((course: Course) => {
                 this.course = Course.from(course);
             }),
-            catchError((response: HttpErrorResponse) => {
-                this.onError(response.message);
+            catchError((error: HttpErrorResponse) => {
+                onError(this.jhiAlertService, error);
                 return of(null);
             }),
         );
```

Comment:
Further down this file in line 90 theres another onError

Reply:
That method uses a string.